RestaurantReservation.Db/DbContext/RestaurantReservationDbContext.cs

```diff
@@ -0,0 +1,186 @@
+                new Reservation { ReservationId = 3, CustomerId = 3, RestaurantId = 3, TableId = 4, ReservationDate = new DateTime(2024, 4, 14, 20, 0, 0), PartySize = 6 },
+                new Reservation { ReservationId = 4, CustomerId = 4, RestaurantId = 4, TableId = 5, ReservationDate = new DateTime(2024, 4, 15, 18, 30, 0), PartySize = 8 },
+                new Reservation { ReservationId = 5, CustomerId = 5, RestaurantId = 5, TableId = 2, ReservationDate = new DateTime(2024, 4, 16, 19, 30, 0), PartySize = 10 }
+            );
+            
+            // Seed Orders
+            builder.Entity<Order>().HasData(
+                new Order { OrderId = 1, ReservationId = 1, EmployeeId = 1, OrderDate = new DateTime(2024, 4, 12, 18, 15, 0), TotalAmount = 50.00m },
+                new Order { OrderId = 2, ReservationId = 2, EmployeeId = 2, OrderDate = new DateTime(2024, 4, 13, 19, 15, 0), TotalAmount = 30.00m },
+                new Order { OrderId = 3, ReservationId = 3, EmployeeId = 3, OrderDate = new DateTime(2024, 4, 14, 20, 15, 0), TotalAmount = 100.00m },
+                new Order { OrderId = 4, ReservationId = 4, EmployeeId = 4, OrderDate = new DateTime(2024, 4, 15, 18, 45, 0), TotalAmount = 80.00m },
+                new Order { OrderId = 5, ReservationId = 5, EmployeeId = 5, OrderDate = new DateTime(2024, 4, 16, 19, 45, 0), TotalAmount = 120.00m }
+            );
+            
+            // Seed OrderItems
+            builder.Entity<OrderItem>().HasData(
+                new OrderItem { OrderItemId = 1, OrderId = 1, ItemId = 1, Quantity = 2 },
+                new OrderItem { OrderItemId = 2, OrderId = 1, ItemId = 2, Quantity = 1 },
+                new OrderItem { OrderItemId = 3, OrderId = 2, ItemId = 3, Quantity = 1 },
+                new OrderItem { OrderItemId = 4, OrderId = 3, ItemId = 4, Quantity = 3 },
+                new OrderItem { OrderItemId = 5, OrderId = 4, ItemId = 5, Quantity = 2 }
+            );
+            builder.Entity<ReservationCustomerRestaurantView>()
+                .HasNoKey()
+                .ToView("ReservationCustomerRestaurantView");
+            builder.Entity<EmployeeRestaurantView>()
+                .HasNoKey()
+                .ToView("EmployeeRestaurantView");
+        }
+    }
+
+    
+    public class ReservationCustomerRestaurantView {
+        public int ReservationId { get; set; }
+        public DateTime ReservationDate { get; set; }
+        public int PartySize { get; set; }
+        public string CustomerFullName { get; set; }
+        public string RestaurantName { get; set; }
+    }
+
+    public class EmployeeRestaurantView {
+        public int EmployeeId { get; set; }
+        public string EmployeeName { get; set; }
+        public string Position { get; set; }
+        public string RestaurantName { get; set; }
+    }
```

Comment:
move to its own files

Reply:
instead of 'ReservationCustomerRestaurantView ' 
you can go with ReservationView